xcode/Subconscious/Shared/Components/Notebook/Notebook.swift

```diff
@@ -821,7 +772,10 @@ struct NotebookModel: ModelProtocol {
 
         /// Find all instances of this model in the stack and update them
         model.details = state.details.map({ (detail: DetailOuterModel) in
-            guard detail.address.slug == receipt.to.slug else {
+            guard let address = detail.address else {
```

Comment:
Fixed logic that was causing bug with renaming. We should be comparing addresses, not slugs, for equality.

Reply:
Maybe I'm missing something, but it looks like we're still checking slugs on the line below?